src/components/core/Dashboard/UserSide/MyBookings/ModernBookingFlow.tsx

```
@@ -78,7 +78,7 @@ const ModernBookingFlow: React.FC<ModernBookingFlowProps> = ({
   const [selectedTime, setSelectedTime] = useState<string>('');
   const [datePage, setDatePage] = useState(0);
   const [loadingMoreSlots, setLoadingMoreSlots] = useState(false);
-  const [showDebugger] = useState(false);
+  const [] = useState(false);
```

Biome (2.1.2)

[error] Unexpected empty array pattern.



(lint/correctness/noEmptyPattern)